cpp/include/cudf/types.hpp

```diff
@@ -224,6 +225,18 @@ enum class type_id : int32_t {
   NUM_TYPE_IDS  ///< Total number of type ids
 };
 
+/**
+ * @brief Function for determining if `type_id` is fixed_point
+ *
+ * @param id The `type_id` to check
+ * @return true If id is either DECIMAL32 or DECIMAL64
+ * @return false If id is not either DECIMAL32 or DECIMAL64
+ */
+constexpr inline bool is_fixed_point_type_id(type_id id)
```

Comment:
This should go in `traits.hpp` with the similar functions. 
```suggestion
constexpr inline bool is_fixed_point(type_id id)
```

Reply:
Not sure I can use it in `types.hpp` if I move it to `traits.hpp`. Should I do that and then just replicate the logic in `types.hpp`?

Reply:
Done.